src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -10690,6 +10786,30 @@ definitions:
         $ref: '#/definitions/PermissionsInfo'
       definition:
         $ref: '#/definitions/Definition'
+  calculateSizeResponse:
+    type: object
+    properties:
+      id:
+        type: string
+        description: Provides the id for which POST/calculateSize has been initiated.
+      size:
+        type: string
+        description: Provides a folder size in bytes.
+      calculatedAtTime:
+        type: string
+        description: Provides the time when the calculating folder size will be completed.
+      numberOfFiles:
+        type: integer
+        description: Provides the number of files in the folder.
+      status:
+        type: string
+        description: Provides a status that signifies the completion of the calculateSize mechanism.
```

Comment:
This would be a better place to include the list of possible states.

Reply:
We've refined the preceding message along with possible states.